### ISSUE TITLE
Geometry attributes/values

### ISSUE THREAD

**Issue by sphamba** [ltelab/tstore#19]

**Comment by martibosch** [comment]
If I understood correctly, this is automatically done if a non-None `geometry` is provided after #58, right @sphamba ?

Basically if `df_attrs` is a `GeoDataFrame`, we are thus using `GeoDataFrame.to_parquet`, that already takes care of writing the geometry columns. The issue is whether we should use "geoarrow" instead of the default "WKB" (see the `geometry_encoding` argument at https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoDataFrame.to_parquet.html). Any ideas on this @ghiggi ? In any case, I doubt this is any performance bottleneck for most use cases and I would like to avoid any premature optimization, but I can (i) test the effects with a proper benchmark and (ii) add a `to_parquet_kwargs` argument to the `to_tstore` methods which is forwarded to `df_attrs.to_parquet`.

**Comment by sphamba** [comment]
@martibosch Yes, loading a TSDF (or converting into a TSDF) with geometry data will create a TSDF wrapper around a Geopandas dataframe. For writing geometry attributes, we are indeed using `GeoDataFrame.to_parquet`, and for reading, we attempt reading the attributes dataframe with `geopandas.read_parquet`, which fails if there is no geometry attributes.
